### PR TITLE
Code snippets used in client projects

### PR DESCRIPTION
**Pull request by fousa**
🔥Please merge #16 first.

### CODE REVIEW

File: Sources/Extensions/UIView+Constraints.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+//  Keychain
+//
+//  Created by Jelle Vandebeeck on 11/02/2018.
+//
+
+import UIKit
+
+extension UIView {
+    
+    public func constraint(to view: UIView) {
```
Comment (herre) [inline Sources/Extensions/UIView+Constraints.swift]: 🎈would name this with a verb
`func constrain(to view: UIView)` or `func constrainToEdgesOf(_ view: UIView)`

File: Sources/Extensions/UIViewController+Child.swift [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+//  Created by Jelle Vandebeeck on 11/02/2018.
+//
+
+import UIKit
+
+extension UIViewController {
+    
+    /// Add a given controller to the current controller as a child view controller. The given
+    /// controller's view is added as a subview to the given `UIView`.
+    public func add(controller: UIViewController, to view: UIView) {
```
Comment (herre) [inline Sources/Extensions/UIViewController+Child.swift]: 🎈would call the argument `viewController` iso `controller`, but seems like 🍎 themselves aren't really consistent in that naming (eg `func addChildViewController(_ `**childController**`: UIViewController)` vs `func overrideTraitCollection(`**forChildViewController childViewController**`: UIViewController) -> UITraitCollection?`)

File: Sources/Extensions/UIViewController+Child.swift [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        // Configure Child View
+        controller.view.constraint(to: view)
+        
+        // Notify Child View Controller
+        controller.didMove(toParentViewController: self)
+    }
+    
+    /// Remove the given controller as a child from the current controller. It's view will
+    /// also be removed as a subview.
+    public func remove(controller: UIViewController) {
```
Comment (herre) [inline Sources/Extensions/UIViewController+Child.swift]: 🎈idem cfr above

File: Sources/Extensions/UIView+Constraints.swift (hunk 1 above)
Comment (fousa) [reply Sources/Extensions/UIView+Constraints.swift]: Or `func constrainToEdges(of view: UIView)`?

**Pull request merged**